fmriprep/config.py

```diff
@@ -304,6 +304,9 @@ def init(cls):
             }
         })
 
+        if cls.omp_nthreads is None:
+            cls.omp_nthreads = max(1, min(cls.nprocs - 1 if cls.nprocs > 1 else os.cpu_count(), 8))
```

Comment:
will this ever be 1 (is the outer max necessary)?

Reply:
Oh, maybe not. I think the if/else avoids it.